tests/assets/testkey.txt

```diff
@@ -0,0 +1,5 @@
+abc def ghi jkl mno pqr stu vwx yz1@3$
```

Comment:
Here are a couple of examples:
* phpunit: https://github.com/mradcliffe/xero/blob/8.x-1.x/tests/src/XeroClientFactoryTest.php#L72
* simpletest: https://api.drupal.org/api/drupal/core!modules!system!src!Tests!File!FileTestBase.php/function/FileTestBase%3A%3AcreateUri/8

It's just important that tearDown() deletes the file, and I think in that example i probably should have created it in /tmp instead of just module directory. From drupal api,